Packs/SMIME_Messaging/Integrations/SMIME_Messaging/SMIME_Messaging.py

```diff
@@ -173,112 +499,164 @@ def decode_str(decrypted_text: bytes, encoding: str) -> Tuple[str, str]:
     return out, msg
 
 
-def decrypt_email_body(client: Client, args: Dict, file_path=None):
+def decrypt_email_body(client: Client, args: dict):
     """ Decrypt the message
 
     Args:
         client: Client
         args: Dict
-        file_path: relevant for the test module
     """
-    if file_path:
-        encrypt_message = file_path
+    if 'test_file_path' in args:  # test module
+        encrypt_message = {'path': args.get('test_file_path', '')}
     else:
-        encrypt_message = demisto.getFilePath(args.get('encrypt_message'))
+        encrypt_message = demisto.getFilePath(args.get('encrypt_message', ''))
+        demisto.debug(f'\n\nFile Name:{encrypt_message["name"]}; Type:{type(encrypt_message["name"])}\n\n')
```

Comment:
```suggestion
        demisto.debug(f'File Name:{encrypt_message["name"]}; Type:{type(encrypt_message["name"])}')
```
I don't think the /n are needed